src/components/marketing/logo/logo.js

```diff
@@ -2,7 +2,7 @@ import xs from 'xstream'
 import { img } from '@cycle/dom'
 
 const view = () => xs.of(
-  img('.logo', { props: { src: '/logo-web2day.png' } }),
+  img('.logo', { props: { src: '/logo.png' } }),
```

Comment:
Where is the logo.png ?

Reply:
i wanted to put alakarte logo, waht do you think ?

Reply:
Yes go ahead